lib/publify/api.rb

```diff
@@ -0,0 +1,18 @@
+require 'net/http'
+
+module Publify
+  class API
+
+    def self.latest_links(limit)
+      begin
+        uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+        response = Net::HTTP.get uri
+        JSON.parse(response).first(limit)
+      rescue Exception => e
+        Rails.logger.error(e.message)
+        []
+      end
+    end
+
+  end
+end
```

Comment:
Humm, I know there are just one resource for now, but I prefer organize this in a resource way (e.g):

```ruby
   module Publify
     class Article
       include ActiveModel::Model
       attr_accessor :title, :link
       def self.resource_name
         'articles.json'
       end

       def self.latest(limit)
         # run_request(resource_name).map { |article_hash| new(article_hash) }
       end
     end
   end
```

You get the idea. Wdyt?